src/scripts/components/form.js

```diff
@@ -0,0 +1,16 @@
+export default function Form() {
+  const form = document.querySelector('.contacts_form');
+
+  form.onsubmit = (event) => {
+    event.preventDefault();
+
+    const formData = new FormData(form);
+
+    for (let [name, value] of formData.entries()) {
+      console.log(name, ':', value);
+    }
+
+    form.reset();
+    window.scrollTo(0, 0);
+  };
+}
```

Comment:
You can not export form, and write all in a global scope (as alternative)
```suggestion

  const form = document.querySelector('.contacts_form');

  form.onsubmit = (event) => {
    event.preventDefault();

    const formData = new FormData(form);

    for (let [name, value] of formData.entries()) {
      console.log(name, ':', value);
    }

    form.reset();
    window.scrollTo(0, 0);
  };
```